.storybook/mixins/apollo-story-mixin.js

```diff
@@ -1,13 +1,35 @@
 export default {
 	provide: {
-		apollo: {
+		federation: {
```

Comment:
Why is there a split on federation. Seems like this should just always use the public/prod Federation Graphql endpoint...

Reply:
We don't currently reach out to any endpoints in storybook. Just mocks. 

GlobalPromotionBannerContentful referenced federation and was causing storybook to barf. 
https://github.com/kiva/ui/blob/master/src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue#L41

Reply:
Wait that's not right...

Reply:
@mcstover Would it be better to change GlobalPromotionBannerContentful.vue to reference apollo instead of federation?

Reply:
cc: @eddieferrer

Reply:
ahh ok that makes sense. a little future clean up then... ty.

Reply:
No big rush to change that atm.

Reply:
@ryan-ludwig I agree with this approach for the stories. the federation provider might be able to be refactored or removed at this point, but that is outside the scope of this work. On the other hand, it is nice to be able to hit the federation server separately while in dev...